### PR TITLE
FIX: Fix the AI's madness

### PR DESCRIPTION
**Pull request by Thomaltarix**
I fixed the AI's madness.

When playing, having only moves that have negative win rates can happen.

In case it happened, the AI did this: 
image

Here, the AI was playing in the top left corner because a default value was set with a win rate equal to 0.
But if the possible moves win rates are all negative, it's the better one, so it was chosen.

### CODE REVIEW

File: src/ai.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self, map_size : int = 0) -> None:
         :param map_size: the size of the map (default is 0)
         """
         self.__map : [[int]] = [[0 for x in range(map_size)] for y in range(map_size)]
-        self.__best_move : Move = Move(-1, -1, 0)
+        self.__best_move : Move = Move(-1, -1, -200000)
```
Comment (Peralban) [inline src/ai.py:32]: image

**Review by Peralban — APPROVED** [review]

File: src/ai.py (hunk 1 above)
Comment (AubaneNourry) [reply src/ai.py:32]: :+1:

**Review by AubaneNourry — APPROVED** [review]

**Review by Peralban — APPROVED** [review]
-200000 So random wtf

File: src/ai.py (hunk 1 above)
Comment (Thomaltarix) [reply src/ai.py]: Fixed

**Review by Peralban — APPROVED** [review]

**Review by AubaneNourry — APPROVED** [review]

**Pull request merged**